src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckNullCaseLabel.java

```diff
@@ -0,0 +1,75 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java21
+package com.puppycrawl.tools.checkstyle.checks.coding.missingswitchdefault;
+
+public class InputMissingSwitchDefaultCheckNullCaseLabel {
```

Comment:
Somehow we lost exact code from the issue, this is most critical input: https://github.com/checkstyle/checkstyle/issues/14068#issue-2012342220

Reply:
Added original test case @nrmancuso